README.md

```diff
@@ -29,3 +29,5 @@ Then you can view the documentation with:
 - google-chrome build/html/index.html
 
 or replace google-chrome with your favorite browser.
+
+Otherwise, here is the link to the online [__documentation__](https://watts.readthedocs.io/en/latest/user/index.html).
```

Comment:
Now that we have online documentation, I think most people won't need to build the documentation locally, so perhaps we should just change this remove the instructions for building locally.

Reply:
Right I changed the order to provide first the link, then the option to generate the documentation locally.